frontend/src/features/properties/map/acquisition/add/models.ts

```diff
@@ -41,6 +47,14 @@ export class AcquisitionForm {
           acquisitionFile: { id: this.id },
         };
       }),
+      acquisitionTeam: this.team
+        .filter(x => !!x.contact && !!x.contactTypeCode)
+        .map<Api_AcquisitionFilePerson>(x => {
+          return {
+            personId: x.contact?.personId || 0,
+            personProfileTypeCode: x.contactTypeCode,
+          };
+        }),
```

Comment:
This logic could be in a class similar to this one instead of just an interface.

Reply:
done, added new class